sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/ItemList.kt

```diff
@@ -0,0 +1,25 @@
+package co.nimblehq.sample.compose.ui.screens.home
+
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.material.Divider
+import androidx.compose.runtime.Composable
+import co.nimblehq.sample.compose.model.UiModel
+
+@Composable
+fun ItemList(
+    uiModels: List<UiModel>,
+    onItemClick: (UiModel) -> Unit
+) {
+    LazyColumn {
+        itemsIndexed(items = uiModels) { index, uiModel ->
+            Item(
+                uiModel = uiModel,
+                onClick = onItemClick
+            )
+
+            val lastItemIndex = uiModels.size - 1
+            if (index != lastItemIndex) Divider()
```

Comment:
I think we can provide the index computation directly into the if check for simplicity. What do you think?
```suggestion
            if (index != uiModels.size - 1) Divider()
```

Reply:
Also, I think we can use just `if (index != uiModels.lastIndex) Divider()` So we get rid of using `- 1` too 🙏